objective-c/Gherkin/GHHasTagsProtocol.h

```diff
@@ -1,4 +1,5 @@
-﻿@class GHTag;
+﻿#import <Foundation/Foundation.h>

```

Comment:
Are all of these `#import <Foundation/Foundation.h>` changes needed? Would it be possible to make a change with more minimal modifications - just the minimum necessary?